pkg/workceptor/workceptor.go

```diff
@@ -631,6 +636,11 @@ func (w *Workceptor) scanForUnits() {
 				}
 				w.activeUnits[fi.Name()] = unit
 				if si.Node != "" && si.Node != w.nc.NodeID() {
+					if si.LocalCancel {
+						remoteNodeID := si.Node
+						remoteUnitID := si.RemoteUnitID
+						go w.attemptRemoteAction(remoteNodeID, remoteUnitID, "cancel", unit)
+					}
 					go w.monitorRemoteUnit(unit, fi.Name())
```

Comment:
even if LocalCancel is true, we still want to monitor the remote node until we get a WorkStateFailed from the remote status, thus we run the monitor even if we've issued the cancel.